sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/EventApiService.java

```diff
@@ -41,9 +39,28 @@ public Event buildGeneratedEvent() {
         .srcType(SourceTypeValues.getRandomSourceTypeName())
         .eventInvestigationStatus("PENDING")
         .eventTitle(String.valueOf(System.currentTimeMillis()))
+        .eventDesc("Event description")
         .startDate(new Date())
         .reportDateTime(new Date())
-        .eventLocation(EventLocation.builder().uuid(UUID.randomUUID().toString()).build())
+        // .eventLocation(EventLocation.builder().uuid(UUID.randomUUID().toString()).build())
+        .eventLocation(
```

Comment:
before adding this new sub-pojo, please check also another test using this entity, so we can make sure that it won't cause any failures.
Find any test using it, run it on local and post a passing proof.

Reply:
ok, thank you